### PR TITLE
Complete bash completions for skopeo

### PR DESCRIPTION
**Pull request by rhatdan**
Current code only handled commands not the options.

### CODE REVIEW

File: completions/bash/skopeo [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+     local options_with_args="
+     "
+     local boolean_options="
+     "
+    _complete_ "$options_with_args" "$boolean_options"
+}
+
+_skopeo_skopeo() {
+     local options_with_args="
+	   --username
```
Comment (runcom) [inline completions/bash/skopeo]: these won't be here after #254

Comment (rhatdan) [inline completions/bash/skopeo]: Ok, I will remove them once I have this merged.

Comment (runcom) [inline completions/bash/skopeo]: > Ok, I will remove them once I have this merged.

Do you want this PR merged first? cause I already merged #254 :smile:

**Comment by rhatdan** [comment]
A little script I wrote to check for options:
```
 for i in $(./skopeo --generate-bash-completion); do echo "========= $i ============"; ./skopeo $i --help; done
```

**Comment by rhatdan** [comment]
Updated to match https://github.com/projectatomic/skopeo/pull/254

**Comment by runcom** [comment]
LGTM

**Comment by runcom** [comment]
@rhatdan could you rebase this one and I'll merge?

**Comment by rhatdan** [comment]
Rebased.

**Pull request merged**